src/form/editors/impl/radioGroup/templates/radioButton.hbs

```diff
@@ -1 +1,6 @@
-<span class="radiobutton js-toggle-button"></span> <span class="radiobutton-text js-display-text">{{displayText}}</span>
\ No newline at end of file
+<span class="radiobutton js-toggle-button"></span>
+{{#if displayHTML}}
+{{{displayHTML}}}
+{{else}}
+<span class="radiobutton-text js-display-text">{{displayText}}</span>

```

Comment:
You should keep the `span` in both cases since we don't imply any restrictions on displayHtml. `{{displayText}}` is what we replace.